modules/azure_k8s_base/azure-k8s-base.json

```diff
@@ -23,6 +23,21 @@
       "description": "Additional configuration for nginx ingress. [Available options](https://kubernetes.github.io/ingress-nginx/user-guide/nginx-configuration/configmap/#configuration-options)",
       "default": {}
     },
+    "cert_manager_values": {
+      "type": "object",
+      "description": "Certificate Manager helm chart additional values. [Available options](https://artifacthub.io/packages/helm/cert-manager/cert-manager?modal=values)",
+      "default": {}
+    },
+    "linkerd_values": {
+      "type": "object",
+      "description": "Linkerd helm chart additional values. [Available options](https://artifacthub.io/packages/helm/linkerd2/linkerd2/2.10.2?modal=values)",
```

Comment:
this creates one more thing to maintain when we update the linkerd version. You can try to define the version in a constant - and pass that version as a terraform input variable.

not blocking for this PR though.

Reply:
I know this will create some additional works, as and when we upgrade the versions, but this enabled users to use the chart values of the specific version, and not make some unintentional mistakes.